src/cpp/meson.build

```diff
@@ -0,0 +1,66 @@
+message('---------- And now the CPP part ----------')
+extra_cpp_args = ['-I../src/samtools', '-DNDEBUG']
```

Comment:
`include_directories()`
**Never** add `-DNDEBUG`, Meson adds it in release mode